vdf/src/gmp_classgroup/congruence.rs

```diff
@@ -36,6 +45,13 @@ impl Default for CongruenceContext {
     }
 }
 
+/// Solves `a*x = b (mod m)`, storing `x` in `mu`
+///
+/// This function may clobber any or all of `self`’s member variables.
+///
+/// # Panics
+///
+/// Panics if the congruence could not be solved.
```

Comment:
Should that comment be on `solve_linear_congruence` instead of `CongruenceContext`?

Reply:
Fixed